server/server.go

```diff
@@ -256,3 +262,188 @@ func (r *rodeServer) ListOccurrences(ctx context.Context, occurrenceRequest *pb.
+	log := r.logger.Named("GetPolicy")
+
+	search := &esSearch{
+		Query: &filtering.Query{
+			Term: &filtering.Term{
+				"id.keyword": getPolicyRequest.Id,
+			},
+		},
+	}
+
+	policy := &pb.Policy{}
+	encodedBody, requestJson := encodeRequest(search)
+	res, err := r.esClient.Search(
+		r.esClient.Search.WithContext(ctx),
+		r.esClient.Search.WithIndex(rodeElasticsearchPoliciesIndex),
+		r.esClient.Search.WithBody(encodedBody),
+	)
+	log = log.With(zap.String("request", requestJson))
+
+	if err != nil {
+		return nil, createError(log, "error sending request to elasticsearch", err)
+	}
+	if res.IsError() {
+		return nil, createError(log, "error searching elasticsearch for document", nil, zap.String("response", res.String()), zap.Int("status", res.StatusCode))
+	}
+
+	var searchResults esSearchResponse
+	if err := decodeResponse(res.Body, &searchResults); err != nil {
+		return nil, createError(log, "error unmarshalling elasticsearch response", err)
+	}
+
+	if searchResults.Hits.Total.Value == 0 {
+		log.Debug("document not found", zap.Any("search", search))
+		return nil, status.Error(codes.NotFound, fmt.Sprintf("%T not found", policy))
+	}
+	protojson.Unmarshal(searchResults.Hits.Hits[0].Source, proto.MessageV2(policy))
+
+	return policy, nil
+
+}
+
+func (r *rodeServer) DeletePolicy(ctx context.Context, deletePolicyRequest *pb.DeletePolicyRequest) (*emptypb.Empty, error) {
+	log := r.logger.Named("DeletePolicy")
+
+	search := &esSearch{
+		Query: &filtering.Query{
+			Term: &filtering.Term{
+				"id.keyword": deletePolicyRequest.Id,
+			},
+		},
+	}
+
+	encodedBody, requestJSON := encodeRequest(search)
+	log.Debug("es request payload", zap.Any("payload", requestJSON))
+
+	res, err := r.esClient.DeleteByQuery(
+		[]string{rodeElasticsearchPoliciesIndex},
+		encodedBody,
+		r.esClient.DeleteByQuery.WithContext(ctx),
+	)
+	if err != nil {
+		return nil, createError(log, "error sending request to elasticsearch", err)
+	}
+	if res.IsError() {
+		return nil, createError(log, "received unexpected response from elasticsearch", nil)
+	}
+
+	var deletedResults esDeleteResponse
+	if err = decodeResponse(res.Body, &deletedResults); err != nil {
+		return nil, createError(log, "error unmarshalling elasticsearch response", err)
+	}
+
+	if deletedResults.Deleted == 0 {
+		return nil, createError(log, "elasticsearch returned zero deleted documents", nil, zap.Any("response", deletedResults))
+	}
+
+	return &emptypb.Empty{}, nil
+}
+
+func (r *rodeServer) ListPolicies(ctx context.Context, listPoliciesRequest *pb.ListPoliciesRequest) (*pb.ListPoliciesResponse, error) {
+	log := r.logger.Named("List Policies")
+
+	// filtering logic
+
+	// encodedBody, requestJSON := encodeRequest(search)
+	// log.Debug("es request payload", zap.Any("payload", requestJSON))
+	var policies []*pb.Policy
+	res, err := r.esClient.Search(
+		r.esClient.Search.WithContext(ctx),
+		r.esClient.Search.WithIndex(rodeElasticsearchPoliciesIndex),
```

Comment:
we need to specify a max page size, or this will only return a max of 10 results, similar to https://github.com/rode/grafeas-elasticsearch/blob/c072ef5eb2ff4f2f1294d281d9158f5e210111ea/go/v1beta1/storage/elasticsearch.go#L873